sax/src/main/java/feign/sax/SAXDecoder.java

```diff
@@ -74,6 +74,10 @@ public Object decode(Response response, Type type) throws IOException, DecodeExc
       XMLReader xmlReader = XMLReaderFactory.createXMLReader();
       xmlReader.setFeature("http://xml.org/sax/features/namespaces", false);
       xmlReader.setFeature("http://xml.org/sax/features/validation", false);
+      xmlReader.setFeature("http://xml.org/sax/features/external-general-entities", false);
```

Comment:
Add a comment above your changes.. "Explicitly control sax configuration to prevent XXE attacks"